core/src/validator.rs

```diff
@@ -119,23 +120,12 @@ impl Validator {
         warn!("vote pubkey: {:?}", vote_account);
         warn!("CUDA is {}abled", if cfg!(cuda) { "en" } else { "dis" });
         info!("entrypoint: {:?}", entrypoint_info_option);
-        info!("{:?}", node.info);
-        info!(
-            "local gossip address: {}",
-            node.sockets.gossip.local_addr().unwrap()
-        );
-        info!(
-            "local broadcast address: {}",
-            node.sockets.broadcast.local_addr().unwrap()
-        );
-        info!(
-            "local repair address: {}",
-            node.sockets.repair.local_addr().unwrap()
-        );
-        info!(
-            "local retransmit address: {}",
-            node.sockets.retransmit.local_addr().unwrap()
-        );
+
+        Self::print_node_info(&node);
+
+        info!("Initializing cuda, this could take a while...");
```

Comment:
Perhaps rename "Initializing cuda" to "Initializing sigverify", to avoid confusing when cuda is disabled?